config/testconfig.go

```diff
@@ -39,3 +49,380 @@ type TestConfig struct {
 	Seth                   *seth.Config           `toml:"Seth"`
 	NodeConfig             *NodeConfig            `toml:"NodeConfig"`
 }
+
+// Read config values from environment variables
+func (c *TestConfig) ReadConfigValuesFromEnvVars() error {
```

Comment:
It'd be good to reflect this function in documentation so people understand they need to add their new env vars there.

Reply:
@skudasov QA will not have to add env vars here unless they add a secret.

Reply:
added doc in https://github.com/smartcontractkit/chainlink-testing-framework/pull/1020